code/modules/mob/living/carbon/human/species.dm

```diff
@@ -958,9 +959,17 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 /datum/species/proc/check_species_weakness(obj/item, mob/living/attacker)
 	return 0 //This is not a boolean, it's the multiplier for the damage that the user takes from the item.It is added onto the check_weakness value of the mob, and then the force of the item is multiplied by this value
 
+/*
+ * Soft equip the outfit required for life. Soft equip will equip if the slot is free, or drop to floor.
+ */ 
+/datum/species/proc/give_important_for_life(mob/living/carbon/human/human_to_equip)
+	if(outfit_important_for_life)
```

Comment:
this is just a tip in general and a bit nitpicky so you dont have to fix this, but doing if(!outfit_important_for_life) and then returning if true prevents nesting.

Reply:
And try and personally avoid early returns if reasonable.

Reply:
why

Reply:
i just checked, it's a contribution guideline as well so for things like this specifically its basically mandated https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-early-return

Reply:
Why was this resolved?

Reply:
my bad, meant to come back to this